cmd/utils.go

```diff
@@ -318,3 +325,89 @@ func isURLContains(srcURL, tgtURL, sep string) bool {
 	}
 	return false
 }
+
+// Return the file system attribute as string; containing mode,
+// uid, gid, uname, Gname, atime, mtime, ctime and md5
+func getFileSystemAttrs(file string) (string, error) {
+	var uname, gname, atime, mtime, ctime, mode, uid, gid string
+	fi, err := os.Stat(file)
+	if err != nil {
+		return "", err
+	}
+	st := fi.Sys().(*syscall.Stat_t)
+	mode = fmt.Sprint(st.Mode)
+	uid = fmt.Sprint(st.Uid)
+	gid = fmt.Sprint(st.Gid)
+
+	i, err := user.LookupId(strconv.FormatUint(uint64(st.Uid), 10))
+	if err != nil {
+		return "", err
+	}
+	uname = i.Username
+
+	g, err := user.LookupGroupId(strconv.FormatUint(uint64(st.Gid), 10))
+	if err != nil {
+		return "", err
+	}
+	gname = g.Name
+
+	t, err := times.Stat(file)
```

Comment:
why do we need this call @sinhaashish we can access the necessary ModTime and AccessTime all with in the same os.Stat() call, I don't think we need to depend on new projects right now.

Reply:
https://golang.org/pkg/syscall/#Stat_t already has Atim captured it shall be just `Atim.Sec`